src/utils/leaderboard.ts

```diff
@@ -0,0 +1,26 @@
+export const calcPayout = (rank: number) => {
```

Comment:
This more likely will be disabled until further notice.

Reply:
It's in the designs so I put it there. Should I just yeet the column out?